mmv1/products/integrationconnectors/Connection.yaml

```
@@ -0,0 +1,1120 @@
+              This private key will be used to sign JWTs used for the jwt-bearer authorization grant.
+              Specified in the form as: projects/*/secrets/*/versions/*.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: "secretVersion"
+                required: true
+                description: |
+                  The resource name of the secret version in the format,
+                  format as: projects/*/secrets/*/versions/*.
+          - !ruby/object:Api::Type::NestedObject
+            name: "jwtClaims"
+            description: |
+              JwtClaims providers fields to generate the token.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: "issuer"
+                description: |
+                  Value for the "iss" claim.
+              - !ruby/object:Api::Type::String
+                name: "subject"
+                description: |
+                  Value for the "sub" claim.
+              - !ruby/object:Api::Type::String
+                name: "audience"
+                description: |
+                  Value for the "aud" claim.
+        exactly_one_of:
+          - auth_config.0.user_password
+          - auth_config.0.oauth2_jwt_bearer
+          - auth_config.0.oauth2_client_credentials
+          - auth_config.0.ssh_public_key
+          - auth_config.0.oauth2_auth_code_flow
+      - !ruby/object:Api::Type::NestedObject
+        name: "oauth2ClientCredentials"
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "clientId"
+            description: |
+              Secret version of Password for Authentication.
+          - !ruby/object:Api::Type::NestedObject
+            name: "clientSecret"
+            description: |
+              Secret version reference containing the client secret.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: "secretVersion"
+                required: true
+                description: |
+                  The resource name of the secret version in the format,
+                  format as: projects/*/secrets/*/versions/*.
+        exactly_one_of:
+          - auth_config.0.user_password
+          - auth_config.0.oauth2_jwt_bearer
+          - auth_config.0.oauth2_client_credentials
+          - auth_config.0.ssh_public_key
+          - auth_config.0.oauth2_auth_code_flow
+      - !ruby/object:Api::Type::NestedObject
+        name: "sshPublicKey"
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "username"
+            description: |
+              The user account used to authenticate.
+          - !ruby/object:Api::Type::NestedObject
+            name: "sshClientCert"
+            description: |
+              Secret version reference containing the client secret.
```

```suggestion
              SSH Client Cert. It should contain both public and private key.
```
copied from the API doc

---

Updated